docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+---
+version: "3"
+services:
+  unibocalendar:
+    container_name: "unibocalendar"
+    image: "unibocalendar"
+    restart: unless-stopped
+    ports:
+      - "3000:3000"
+    volumes:
+      - "./db:/app/logs"
+      - "./data:/app/opendata"
```

Comment:
Good, I assume this is to ease the configuration since you already configured the right volumes.

Reply:
Maybe adding a mention on the README could be useful. Also maybe providing also a plain docker command.